services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/attachment/common/AttachmentEntity.kt

```diff
@@ -170,6 +170,8 @@ interface HankeAttachmentRepository : JpaRepository<HankeAttachmentEntity, UUID>
 interface ApplicationAttachmentRepository : JpaRepository<ApplicationAttachmentEntity, UUID> {
     fun findByApplicationId(applicationId: Long): List<ApplicationAttachmentEntity>
 
+    fun findByBlobLocationIsNull(): List<ApplicationAttachmentEntity>
```

Comment:
```suggestion
    fun findFirstByBlobLocationIsNull(): ApplicationAttachmentEntity?
```

Reply:
Thank you for this, I did not know about `First`.